InnerEye/ML/dataset/full_image_dataset.py

```diff
@@ -213,10 +213,11 @@ class FullImageDataset(GeneralDataset):
     """
 
     def __init__(self, args: SegmentationModelBase, data_frame: pd.DataFrame,
-                 full_image_sample_transforms: Optional[Compose3D[Sample]] = None):
+                 full_image_sample_transforms: Optional[Compose3D[Sample]] = None,
+                 allow_incomplete_labels: bool = False):
```

Comment:
why do we have an extra argument allow_imcomplete... here? it is already contained in args.

Reply:
Good point. Fixed.